setup.py

```diff
@@ -1,130 +1,4 @@
-            "lightly-embed = lightly.cli.embed_cli:entry",
-            "lightly-magic = lightly.cli.lightly_cli:entry",
-            "lightly-serve = lightly.cli.serve_cli:entry",
-            "lightly-train = lightly.cli.train_cli:entry",
-            "lightly-version = lightly.cli.version_cli:entry",
-        ]
-    }
-
-    python_requires = ">=3.6"
-    base_requires = load_requirements(filename="base.txt")
-    torch_requires = load_requirements(filename="torch.txt")
-    video_requires = load_requirements(filename="video.txt")
-    dev_requires = load_requirements(filename="dev.txt")
-
-    setup_requires = ["setuptools>=21"]
-    install_requires = base_requires + torch_requires
-    extras_require = {
-        "video": video_requires,
-        "dev": dev_requires,
-        "all": dev_requires + video_requires,
-    }
-
-    packages = setuptools.find_packages(include=["lightly*"])
-
-    project_urls = {
-        "Homepage": "https://www.lightly.ai",
-        "Web-App": "https://app.lightly.ai",
-        "Documentation": "https://docs.lightly.ai",
-        "Github": "https://github.com/lightly-ai/lightly",
-        "Discord": "https://discord.gg/xvNJW94",
-    }
-
-    classifiers = [
-        "Development Status :: 5 - Production/Stable",
-        "Intended Audience :: Developers",
-        "Intended Audience :: Education",
-        "Intended Audience :: Science/Research",
-        "Topic :: Scientific/Engineering",
-        "Topic :: Scientific/Engineering :: Artificial Intelligence",
-        "Topic :: Scientific/Engineering :: Image Processing",
-        "Topic :: Scientific/Engineering :: Mathematics",
-        "Topic :: Software Development",
-        "Topic :: Software Development :: Libraries",
-        "Topic :: Software Development :: Libraries :: Python Modules",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
-        "Programming Language :: Python :: 3.11",
-        "License :: OSI Approved :: MIT License",
-    ]
-
-    setuptools.setup(
-        name=name,
-        version=version,
-        author=author,
-        author_email=author_email,
-        description=description,
-        entry_points=entry_points,
-        license="MIT",
-        license_files=["LICENSE.txt"],
-        long_description=long_description,
-        long_description_content_type="text/markdown",
-        setup_requires=setup_requires,
-        install_requires=install_requires,
-        extras_require=extras_require,
-        python_requires=python_requires,
-        packages=packages,
-        classifiers=classifiers,
-        include_package_data=True,
-        project_urls=project_urls,
-    )
+# for packaging tools not supporting, e.g., PEP 517, PEP 660
```

Comment:
Let's make this comment more clear. When do we need this case and what are the consequences of keeping it? Please also mention why it's not necessary to duplicate all the information that was removed above.

Reply:
It is specified in https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html

Under an "Important" banner

Reply:
Ah, yes. Let's then clarify the comment:
```suggestion
# For compatibility with legacy builds or versions of tools 
# that don’t support certain packaging standards (e.g. PEP 517
# or PEP 660). The configuration is kept in pyproject.toml
```